formats/service_manual_guide/frontend/examples/service_manual_guide.json

```diff
@@ -11,6 +11,7 @@
         "href": "#what-is-it-why-it-works-and-how-to-do-it"
       }
     ],
+    "change_note": "This is our latest change.\n\nThis is the reason for our latest change.",
```

Comment:
I've updated the examples to remove references to reason for change, and modified publisher to just send the change note.